src/databricks/labs/ucx/install.py

```diff
@@ -396,6 +397,10 @@ def run(self):
                 self.create_jobs,
             ],
         )
+        if self._prompts.confirm("Do you want to run assessment workflow after the installation?"):
```

Comment:
this is `WorkspaceInstallation`, i explicitly told to make a change in `WorkspaceInstaller` on fresh installs. e.g. customers don't have run have it run by default when they upgrade

Reply:
Changed the prompt to workspace installer so that it is get called only for fresh installs.